### PR TITLE
feat: (REPLAT-7055) Redesign lead 2no pic and 2 slice for medium breakpoint

### PR DESCRIPTION
**Pull request by pdimova**
Story: https://nidigitalsolutions.jira.com/browse/REPLAT-7055

Screenshot_1564642720
Simulator Screen Shot - iPad Air 2 - 2019-08-01 at 09 58 43L
Screenshot_1564589747
Simulator Screen Shot - iPad Air 2 - 2019-07-31 at 19 20 36

### CODE REVIEW

File: packages/edition-slices/src/tiles/tile-y/index.js [hunk 1]
```diff
@@ -1,38 +1,46 @@
...
-);
+const TileY = ({ onPress, tile, breakpoint = editionBreakpoints.medium }) => {
+  const styles = stylesFactory(breakpoint);
+
+  return (
+    <TileLink
+      onPress={onPress}
+      style={styles.container}
+      tile={tile}
+      // withStar={false}
```
Comment (ValGeorgiev) [inline packages/edition-slices/src/tiles/tile-y/index.js]: Please delete this line if its not needed

File: packages/edition-slices/src/tiles/tile-y/index.js [hunk 2]
```diff
@@ -1,38 +1,46 @@
...
+    >
+      <WithoutWhiteSpace
+        render={whiteSpaceHeight => (
+          <TileSummary
+            headlineStyle={styles.headline}
+            summary={getTileSummary(tile, 800)}
+            tile={tile}
+            // withStar
+            // starStyle={styles.star}
```
Comment (ValGeorgiev) [inline packages/edition-slices/src/tiles/tile-y/index.js]: here too

File: packages/edition-slices/src/tiles/tile-y/index.js (hunk 1 above)
Comment (pdimova) [reply packages/edition-slices/src/tiles/tile-y/index.js]: deleted

File: packages/edition-slices/src/tiles/tile-y/index.js (hunk 2 above)
Comment (pdimova) [reply packages/edition-slices/src/tiles/tile-y/index.js]: my bad

**Pull request merged**